### PR TITLE
updated cart icons to material-ui icons

### PR DESCRIPTION
**Pull request by ItzNesbroDev**
Updated cart ion icons to material ui icon!

### CODE REVIEW

**Review by legendarykamal — CHANGES_REQUESTED** [review]
Please use 
image
for adding icon and 
image
for decreasing on Cart page

**Comment by legendarykamal** [comment]
Also @ItzNesbroDev  replace cross icon with this delete , It will look more finer. 

image

**Review by legendarykamal — CHANGES_REQUESTED** [review]
Try to use red and green colour instead of primary.

**Comment by ItzNesbroDev** [comment]
Ok

**Comment by ItzNesbroDev** [comment]
@legendarykamal Updated!

File: src/App.css [hunk 1]
```diff
@@ -2,222 +2,228 @@
...
-    margin-right: 10px;
-    cursor: pointer;
+.container-products .icon {
+  font-size: 25px;
+  margin-left: 10px;
+  margin-right: 10px;
+  cursor: pointer;
+}
+.container-products .delete {
+  color: rgb(182, 67, 67);
```
Comment (kamalbuilds) [inline src/App.css]: Implementing a darker red colour would be great like using #e32222 for warning

File: src/App.css [hunk 2]
```diff
@@ -2,222 +2,228 @@
...
+}
+.container-products .decrease,
+.increase {
+  color: green;
```
Comment (kamalbuilds) [inline src/App.css:134]: red should be used for decrease #e32222

File: src/components/Cart.js [hunk 3]
```diff
@@ -8,92 +13,84 @@ import rust from "../images/rust.png";
...
+    console.log(item);
+    console.log(basketProps.products[item].inCart);
+    if (basketProps.products[item].inCart) {
+      productsInCart.push(basketProps.products[item]);
+    }
+    console.log(productsInCart);
+  });
 
-    // const productImages = [greyTshirt, greyHoddie, blackTshirt, blackHoddie];
-    const productImages = (product) => {
```
Comment (kamalbuilds) [inline src/components/Cart.js:24]: why are you deleting all this logic and then adding the same below ? probably some prettier change?

**Review by legendarykamal — CHANGES_REQUESTED** [review]
Just a little more changes to get things done and merged :)

File: src/components/Cart.js (hunk 3 above)
Comment (ItzNesbroDev) [reply src/components/Cart.js:24]: it was formatted with prettier when save files

Comment (kamalbuilds) [reply src/components/Cart.js:24]: ok I have filled a new issue to solve this , pls complete the rest and we will be good to go .

Comment (ItzNesbroDev) [reply src/components/Cart.js:24]: Done!

File: src/App.css [hunk 4]
```diff
@@ -2,222 +2,230 @@
...
-    margin-left: 10px;
-    margin-right: 10px;
-    cursor: pointer;
+.container-products .icon {
+  font-size: 25px;
+  margin-left: 10px;
+  margin-right: 10px;
+  cursor: pointer;
+}
+.container-products .delete {
```
Comment (kamalbuilds) [inline src/App.css]: combine .delete and .decrease as these have same properties, rest all looks good .

**Review by legendarykamal — APPROVED** [review]
NIce work :+1:

**Comment by ItzNesbroDev** [comment]
All Done!

File: src/App.css (hunk 4 above)
Comment (kamalbuilds) [reply src/App.css]: @ItzNesbroDev can you delete package-lock.json as its causing some conflicts ? or maybe try to resolve it .

Comment (ItzNesbroDev) [reply src/App.css]: Fixed.

**Comment by ItzNesbroDev** [comment]
@legendarykamal Bro All Done.

File: src/components/Cart.js [hunk 5]
```diff
@@ -8,92 +13,84 @@ import rust from "../images/rust.png";
...
-                <h5 className="total">TOTAL</h5>
-            </div>
-            <div className="products">{productsInCart}</div>
-            <div className="basketTotalContainer">
-                <h4 className="basketTotalTitle">Basket Total</h4>
-                <h4 className="basketTotal">${basketProps.cartCost}.00</h4>
-            </div>
+      <Fragment key={index}>
+        <div className="product">
+          <Delete
```
Comment (kamalbuilds) [inline src/components/Cart.js:49]: For future ref here the new delete icon has been added.

File: src/components/Cart.js [hunk 6]
```diff
@@ -8,92 +13,84 @@ import rust from "../images/rust.png";
...
+            onClick={() => clearProduct(product.tagName)}
+            className="delete icon"
+          />
+          <img src={productImages(product)} />
+          <span className="sm-hide">{product.name}</span>
+        </div>
+        <div className="price sm-hide">${product.price}.00</div>
+        <div className="quantity">
+          <RemoveCircleOutline
```
Comment (kamalbuilds) [inline src/components/Cart.js:58]: dec product - sign

**Review by legendarykamal — APPROVED** [review]
LGTM :medal_sports:

**Comment by legendarykamal** [comment]
congo @ItzNesbroDev You can find your name in the contributors list on the home page :1st_place_medal:

**Pull request merged**